x-pack/test/spaces_api_integration/common/config.ts

```diff
@@ -6,7 +6,8 @@
  */
 
 import path from 'path';
-import { REPO_ROOT } from '@kbn/utils';
+// @ts-ignore we have to check types with "allowJs: false" for now, causing this import to fail
```

Comment:
Wouldn't it be better to use `@ts-expect-error`?

```suggestion
// @ts-expect-error we have to check types with "allowJs: false" for now, causing this import to fail
```